sdk/src/main/scala/com/horizen/account/api/rpc/service/EthService.scala

```diff
@@ -43,24 +45,27 @@ import scala.collection.JavaConverters.seqAsJavaListConverter
 import scala.collection.concurrent.TrieMap
 import scala.collection.convert.ImplicitConversions.`collection AsScalaIterable`
 import scala.collection.mutable.ListBuffer
-import scala.concurrent.duration.{FiniteDuration, SECONDS}
+import scala.concurrent.duration.{FiniteDuration}
 import scala.concurrent.{Await, Future}
 import scala.language.postfixOps
 import scala.util.{Failure, Success, Try}
 
 class EthService(
     scNodeViewHolderRef: ActorRef,
+    networkControllerRef: ActorRef,
     nvtimeout: FiniteDuration,
     networkParams: NetworkParams,
     settings: EthServiceSettings,
+    maxIncomingConnections: Int,
+    rpcClientVersion: String,
```

Comment:
We should separate `EthService` into multiple services, like originally intended. Probably a task for later though.

Reply:
good point, but for now let's keep as is.